assembl/static2/js/app/pages/debate.jsx

```diff
@@ -18,6 +21,7 @@ class Debate extends React.Component {
     this.showThumbnails = this.showThumbnails.bind(this);
     this.hideThumbnails = this.hideThumbnails.bind(this);
     this.displayThumbnails = this.displayThumbnails.bind(this);
+    this.redirectToV1Threads = this.redirectToV1Threads.bind(this);
```

Comment:
You should rename this method to `redirectToV1` and remove the `Threads` term because it's for all phases different than the "survey" phase, not only `thread` but `twoColums` or `tokenVote` for instance.